go.mod

```diff
@@ -7,11 +7,11 @@ require (
 	github.com/shurcooL/graphql v0.0.0-20230722043721-ed46e5a46466
 	github.com/stretchr/testify v1.9.0
 	golang.org/x/oauth2 v0.23.0
+	gopkg.in/yaml.v3 v3.0.1
```

Comment:
Generally we use https://github.com/kubernetes-sigs/yaml which uses json also for yaml fields. I recommend we put gokpg.in/yaml.v3 on our blocking list and use https://github.com/kubernetes-sigs/yaml instead